src/applications/disability-benefits/view-payments/README.md

```diff
@@ -46,187 +46,186 @@ When the front end calls [/profile/payment_history](https://github.com/departmen
     replacing the `payment_history` method in `/app/services/bgs/payment_service.rb` with the following:
 
     ```ruby
-    def payment_history(person)
-        response = {
-            :payments => {
-                :payment => [
+    def payment_history
```

Comment:
The formatting changes are a result of the changes made by the formatter in vets-api, I copied the code from there after I had pasted it from here and let the formatter do its thing